modules/ROOT/pages/policies-compare-mule3-and-mule4.adoc

```diff
@@ -0,0 +1,41 @@
+= Policies and Mule Versions
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: policy, custom, ootb, offline
+
+In Mule versions 3 and 4, the policies remain the same, with a few exceptions. The following section describes the similarities and differences in these Mule versions.
```

Comment:
I suggest replacing this these sentences as follows:

"With the exceptions of <name them here, if there are truly just a few>, the policies for Mule runtime engine versions 3 and 4 are similar."
OR
"Although they are similar, the policies for Mule runtime engine versions 3 and 4 have some differences."

We do not have to refer to "the following section": GLOBALLY avoid textual self-reference in all writing.

Reply:
Added this instead "Policies in Mule versions 3 and differ based on how they are implemented and the level of support provided."